Minor typo fix

tangowithfoxtrot opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description

<!--- Describe your changes in detail here. -->

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

drashna commented:
Could you get the other ploopy readme's too?  

Because this drashna guy is horrible at spelling, and I can pretty much guarantee that it was a copy/paste issue for the readmes.

tangowithfoxtrot commented:
I can. Do you think it'd make more sense to handle it in their repo instead?

drashna commented:
> I can. Do you think it'd make more sense to handle it in their repo instead?

Changing it here would be best, as any changes here will be pulled there, sooner or later.

tangowithfoxtrot commented:
I've updated the remaining Ploopy docs and fixed a few of the broken image links:
https://github.com/qmk/qmk_firmware/pull/14845
https://github.com/qmk/qmk_firmware/pull/14846
https://github.com/qmk/qmk_firmware/pull/14845
